docs/templates/manual.html.j2

```diff
@@ -12,17 +12,43 @@
         <span class="bi bi-list" aria-hidden="true"></span>
       </button>
       <nav id="contents" class="col-md-3 sticky-md-top p-3 overflow-y-auto offcanvas-md offcanvas-end d-print-none" style="top:4.5rem; height:calc(100dvh - 5.5rem);" aria-label="Table of contents">
-        <div class="d-flex justify-content-between">
-          <h4>Contents</h4>
-          <button type="button" class="btn-close d-md-none" data-bs-dismiss="offcanvas" data-bs-target="#contents" aria-label="Close table of contents"></button>
-        </div>
+        <h4>Contents</h4>
+        {%- if toc %}
+        <ul class="ps-0">
+          {%- for section in toc %}
+          {%- if section.level == 1 %}
+          {%- if not loop.first %}
+              </ul>
+            </details>
+          </li>
+          {%- endif %}
+          <li>
+            <details>
+              <summary>
+                <a href="#{{ section.id }}" class="text-decoration-none py-2">{{ section.title }}</a>
+              </summary>
```

Comment:
<img width="317" alt="Screenshot 2024-12-05 at 21 01 37" src="https://github.com/user-attachments/assets/c012266c-658f-451f-a8bd-cefa14db597c">
Triagnle icon is duplicated.

Reply:
Done.